sentry/src/event_reducer.rs

```diff
@@ -1,18 +1,28 @@
-use crate::{payout::get_payout, Session};
+use crate::{
+    payout::{get_payout, PayoutResult},
+    Session,
+};
 use primitives::{
     sentry::{AggregateEvents, Event, EventAggregate},
     BigNum, Channel, ValidatorId,
 };
-use slog::Logger;
+use slog::{info, Logger};
 
 pub(crate) fn reduce(
     logger: &Logger,
     channel: &Channel,
     initial_aggr: &mut EventAggregate,
     ev: &Event,
+    payout: &PayoutResult,
     session: &Session,
 ) -> Result<(), Box<dyn std::error::Error>> {
     let event_type = ev.to_string();
+    if let Ok(Some(payout)) = payout {
+        info!(&logger, "Event type {}, with payout Validator ID: {}, and Pricing of {}", event_type, payout.0, &payout.1.to_string());
+        let cloned_payout = (payout.0, payout.1.clone());
```

Comment:
There's probably a better way to do this. Maybe `PayoutResult` should become its own struct?

Reply:
I am not really certain why do we do have to duplicate logic. The two matches bellow already have the same logic implemented without using `[event_type]` but `initial_aggr.events.get(&event_type)`.
You need to link the payout from the function to the event. See my other comment for using a tuple for example.